index.html

```diff
@@ -971,6 +999,91 @@ <h2>The <code>application/credential-claims-set-1.1+json</code> Media Type</h2>
   <div class="issue" data-number="42"></div>
 </section>
 
+<section>
+  <h2>Appendix</h2>
+  <section>
+    <h2>Example Mapping</h2>
+    <p>
+The following describes a mapping from <code>application/vc+jwt</code> to
+<code>application/vc+ld+json</code>. This is one possible unidirectional mapping
+between 2.0 VC-JWTs and the VC Data Model; other such mappings are possible.
+    </p>
+    <ul>
+      <li>
+Extract <code>iss</code>, <code>sub</code>, <code>iat</code>, <code>nbf</code>,
+<code>exp</code>, <code>jti</code>, and <code>aud</code> as registered claims.
+      </li>
+      <li>
+Set aside all other claims as subject claims.
+      </li>
+      <li>
+To construct the VCDM-compliant JSON:
+        <ul>
+          <li>
+Set the value of <code>@context</code> to
+<code>"https://www.w3.org/ns/credentials/v2"</code>.
+          </li>
+          <li>
+Set the value of <code>type</code> to <code>["VerifiableCredential"]</code>.
+          </li>
+          <li>
+Set the value of the <code>issuer</code> property to one of the following:
+          <ul>
+            <li>
+If the value of <code>iss</code> is a URL, use the value of <code>iss</code>.
+            </li>
+            <li>
+If the value of <code>iss</code> is not a URL, use the concatenation of "<code>urn:vc:</code>"
+and the value of <code>iss</code>.
+            </li>
+          </ul>
+          </li>
+          <li>
+If <code>jti</code> is present, set the value of <code>id</code> to the
+concatenation of "<code>urn:vc:</code>" and the value of <code>jti</code>.
+          </li>
+          <li>
+If <code>nbf</code> is present, set the value of <code>validFrom</code> to the
+<code>dateTime</code> obtained by converting the value of <code>nbf</code> from
+the <code>NumericDate</code> described in RFC 7519 to a <code>dateTime</code> as
+described in XMLSCHEMA11-2.
```

Comment:
Both of these refs need to be turned into ReSpec refs.
```suggestion
the <code>NumericDate</code> described in [[!RFC7519]] to a <code>dateTime</code> as
described in [[!XMLSCHEMA11-2]].
```